src/common/guid.schema.json

```
@@ -0,0 +1,5 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "string",
+  "pattern": "^(\\{{0,1}([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}\\}{0,1})$"
```

Let's get rid of the `{}`'s as that may require additional parsing.

```suggestion
  "pattern": "^(([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12})$"
```